caikit/core/registries.py

```diff
@@ -0,0 +1,75 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Single spot where stick accessors for global shared state
+
+(Shining the light on the shared state so that we can clean it up and re-home it)
+This SHOULD NOT import anything else from caikit outside the toolkit error handler
```

Comment:
Why exactly is this? I agree that we don't want to use other things here, but I'm curious if there's a bigger issue that led to this comment

Reply:
To avoid import cycles, since so many things access these registries. 
These had always been a real pain with imports, which led to the backend registries living in `module.py` instead of with the rest of the backends code. There was a comment saying that they lived there just to avoid import cycles :(

Reply:
Makes sense!